ruby/bowling.rb

```diff
@@ -0,0 +1,48 @@
+class Bowling
+  def initialize
+    @rolls = []
+  end
+
+  def roll(pins)
+    @rolls << pins
+  end
+
+  def score
```

Comment:
Am I right in saying that the same index position in the @rolls array for different instances could contain bowls from different frames? That's a nice way of doing it because you can avoid dealing with the notion of a frame class or other frame distinction. On the other hand, it might make the code harder for other people to interpret.